### ISSUE TITLE
Make Vue a `peerDependency`

### ISSUE THREAD

**Issue by Prinzhorn** [alvarotrigo/vue-fullpage.js#1]
I believe since this project does not actually use vue (e.g. as an `import`) it should be listed as `peerDependency` and not `dependency`. See https://docs.npmjs.com/files/package.json#peerdependencies jQuery, however, should stay as is. I didn't read through all of the code, so I might be wrong though.

**Comment by alvarotrigo** [comment]
Thanks for the suggestion @Prinzhorn !
Feel free to create a pull request if you want!

You suggest the following then?

```json
  "dependencies": {
    "jquery": ">=1.6.0"
  },
  "peerDependencies": {
    "vue": "^2.5.13"
  }
```

**Comment by Prinzhorn** [comment]
Since I'm not sure about the implications I didn't want to create a PR. As far as I can tell this repo does not have any tests and I have never used vue. So I can't make sure nothing breaks ;)

**Comment by alvarotrigo** [comment]
This should now be fixed on the latest version 0.0.2
